recommendations/intro.adoc

```diff
@@ -702,17 +718,35 @@ WITH u1, u1_mean, u2, avg(r.rating) AS u2_mean, ratings
 UNWIND ratings AS r
 
 WITH sum( (r.r1.rating-u1_mean) * (r.r2.rating-u2_mean) ) AS nom,
-     sqrt( sum( (r.r1.rating - u1_mean)^2) * sum( (r.r2.rating - u2_mean) ^2)) AS denom, 
+     sqrt( sum( (r.r1.rating - u1_mean)^2) * sum( (r.r2.rating - u2_mean) ^2)) AS denom,
      u1, u2 WHERE denom <> 0
 
-RETURN u1.name, u2.name, nom/denom AS pearson 
+RETURN u1.name, u2.name, nom/denom AS pearson
 ORDER BY pearson DESC LIMIT 100
 ----
 
+We can also compute this measure using the https://neo4j.com/docs/graph-algorithms/current/algorithms/similarity-pearson[Pearson Similarity algorithm^] in the Neo4j Graph Algorithms Library.
+
+.Find users most similar to Cynthia Freeman, according to the Pearson similarity function
+[source, cypher]
+----
+MATCH (p1:User {name: 'Cynthia Freeman'})-[x:RATED]->(movie:Movie)
+WITH p1, algo.similarity.asVector(movie, x.rating) AS p1Vector
+MATCH (p2:User)-[x:RATED]->(movie:Movie) WHERE p2 <> p1
```

Comment:
should we use x2 here for clarity?  also in the later places and the 2nd list-comprehension?